Common/UnitDefinitions/VolumePerLength.json

```diff
@@ -100,6 +100,18 @@
           "Abbreviations": [ "yd³/ftUS" ]
         }
       ]
+    },
+    {
+      "SingularName": "UsGallonPerMile",
+      "PluralName": "UsGallonsPerMile",
+      "FromUnitToBaseFunc": "{x} / 4.251439077933434e5",
+      "FromBaseToUnitFunc": "{x} * 4.251439077933434e5",
```

Comment:
Based on the two sources below, the functions should be using `425144` exactly.

Reply:
It seems they maybe rounded.


These other two agree on a more precise value:
425143707490521
https://www.unitsconverters.com/en/Gallon(Us)Permile-To-Meterpercubicmeter/Unittounit-4047-4032

4251437074
https://www.advancedconverter.com/other-converters/fuel-consumption/gallons-per-mile-US-to-miles-per-gallon-US

Reply:
Cool! Thanks! Fixed.

I just realized that I'd need these units too, so I added them to this PR:
ImperialGallonPerMile, MegaukGallonPerDay, MegausGallonPerDay

Reply:
Also it would be nice to consolidate Uk vs Imperial prefixes.

Reply:
@angularsen, could you please review this PR?

Reply:
@tmilnthorp, could please you review my PR?

Reply:
> @tmilnthorp, could please you review my PR?

I will try to take a look tomorrow for ya!